spec/tapioca/cli/gem_spec.rb

```diff
@@ -831,74 +831,6 @@ module Foo
           assert_empty(err)
           assert(status)
         end
-
-        it "generates abstract classes properly" do
```

Comment:
Why was this removed?

Reply:
Because it was the wrong location for the test. We already have the same thing tested in `symbol_compiler_spec`, there was no need to test the same thing in a CLI test, and it was causing some problem that I don't remember now, tbh.